miio/integrations/vacuum/roborock/vacuumcontainers.py

```diff
@@ -163,7 +184,32 @@ def error(self) -> str:
             return "Definition missing for error %s" % self.error_code
 
     @property
-    @sensor("Battery", unit="%", device_class="battery")
+    @sensor(
+        "Dock error code",
+        icon="mdi:alert",
+        entity_category="diagnostic",
+        enabled_default=False,
+    )
+    def dock_error_code(self) -> int:
+        """Dock error status as returned by the device."""
+        return int(self.data["dock_error_status"])
```

Comment:
This causes an exception as the key is not available on my v1 vacuum.

Reply:
done